utils/R/convert.input.R

```diff
@@ -5,228 +5,245 @@
       logger.error("input not found", input.id)
       return(NULL)
     }
     
-    dbfile = db.query(paste(
-      "SELECT * from dbfiles where container_id =", input.id, 
-      " and container_type = 'Input' and machine_id =", machine$id), con)
-    if(nrow(dbfile)==0) {
-      logger.error("dbfile not found",input.id);return(NULL)
+    dbfile <- db.query(paste("SELECT * from dbfiles where container_id =", input.id, 
+                             " and container_type = 'Input' and machine_id =", machine$id), con)
+    if (nrow(dbfile) == 0) {
+      logger.error("dbfile not found", input.id)
+      return(NULL)
     }
-    if(nrow(dbfile)>1) {
-      logger.warning("multiple dbfile records, using last",dbfile);
-      dbfile = dbfile[nrow(dbfile),]
+    if (nrow(dbfile) > 1) {
+      logger.warning("multiple dbfile records, using last", dbfile)
+      dbfile <- dbfile[nrow(dbfile), ]
     }
-  } 
-
+  }
+  
   #--------------------------------------------------------------------------------------------------#
-  # Perform Conversion 
-  conversion = "local.remote" #default
+  # Perform Conversion
+  conversion <- "local.remote"  #default
   
-  if(!is.null(browndog) && host$name == 'localhost'){ 
+  if (!is.null(browndog) && host$name == "localhost") {
     # perform conversions with Brown Dog - only works locally right now
-    library(RCurl) 
+    library(RCurl)
     
-    # Determine outputtype using formatname and mimetype of output file
-    # Add issue to github that extension of formats table to include outputtype 
-    if(mimetype ==  'application/x-netcdf') { # Convert to netcdf - only using localhost
-      outputtype <- 'pecan.zip'
-    } else { # Convert to model specific format
-      if(formatname == 'ed.met_driver_header_files_format' ||
-         formatname == 'ed.met_driver_header files format') {
-        outputtype <- 'ed.zip'
-      } else if(formatname == 'Sipnet.climna') {
-        outputtype <- 'clim'
-      } else if(formatname == 'DALEC meteorology') {
-        outputtype <- 'dalec.dat'
-      } else if(formatname == 'LINKAGES met') {
-        outputtype <- 'linkages.dat'
+    # Determine outputtype using formatname and mimetype of output file Add issue to
+    # github that extension of formats table to include outputtype Convert to netcdf
+    # - only using localhost
+    if (mimetype == "application/x-netcdf") {
+      outputtype <- "pecan.zip"
+    } else {
+      # Convert to model specific format
+      if (formatname == "ed.met_driver_header_files_format" || formatname == 
+          "ed.met_driver_header files format") {
+        outputtype <- "ed.zip"
+      } else if (formatname == "Sipnet.climna") {
+        outputtype <- "clim"
+      } else if (formatname == "DALEC meteorology") {
+        outputtype <- "dalec.dat"
+      } else if (formatname == "LINKAGES met") {
+        outputtype <- "linkages.dat"
+      } else {
+        logger.severe(paste("Unknown formatname", formatname))
```

Comment:
Just a reminder to @tonygardella that we've got to slowly work these model-specific hacks out of the code.